crates/spirv-builder/src/test/mod.rs

```diff
@@ -159,6 +159,33 @@ fn dis_fn(src: &str, func: &str, expect: &str) {
     assert_str_eq(expect, &func.disassemble())
 }
 
+fn dis_entry_fn(src: &str, func: &str, expect: &str) {
```

Comment:
I was confused by this so maybe `dis_entry_stub` might be easier to get? A comment would also help, but the name made me think the high-level Rust `fn`, not the generated stub.